### PR TITLE
Feature/task assignee

### PR DESCRIPTION
**Pull request by damm89**

### CODE REVIEW

File: backend/src/zac/camunda/tests/test_set_task_assignee.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+            # data with delegate
+            data.update({"assignee": "", "delegate": user.username})
+            response = self.client.post(
+                self.endpoint,
+                data=data,
+            )
+            self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+            expected_payload = {"userId": user.username}
+            self.assertEqual(m.last_request.json(), expected_payload)
+            self.assertIn("delegate", m.last_request.url)
```
Comment (annashamray) [inline backend/src/zac/camunda/tests/test_set_task_assignee.py:241]: Could you also add a test when `task` is not found?

**Review by annashamray — COMMENTED** [review]
Nice work!

File: backend/src/zac/camunda/api/views.py [hunk 2]
```diff
@@ -233,3 +237,92 @@ def post(self, request, *args, **kwargs):
...
+
+
+class SetTaskAssigneeView(APIView):
+    permission_classes = (permissions.IsAuthenticated & CanPerformTasks,)
+    serializer_class = SetTaskAssigneeSerializer
+
+    def _get_task(self, task_id: str) -> Task:
+        task = get_task(task_id, check_history=False)
+        if task is None:
+            raise exceptions.NotFound(
```
[the remark this answers is not in the document]
Comment (damm89) [reply backend/src/zac/camunda/api/views.py:249]: Yeah tough one. If you think it's better, I'll take your word for it. :)

File: backend/src/zac/camunda/tests/test_set_task_assignee.py (hunk 1 above)
Comment (damm89) [reply backend/src/zac/camunda/tests/test_set_task_assignee.py:241]: You got it!

**Review by annashamray — APPROVED** [review]

File: backend/src/zac/camunda/api/views.py [hunk 3]
```diff
@@ -233,3 +237,93 @@ def post(self, request, *args, **kwargs):
...
+            204: None,
+        },
+    )
+    def post(self, request, *args, **kwargs):
+        serializer = self.serializer_class(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        task_id = serializer.validated_data["task_id"]
+        task = self._get_task(task_id)
+        print("LOL")
```
Comment (sergei-maertens) [inline backend/src/zac/camunda/api/views.py:300]:
```suggestion
```

File: backend/src/zac/camunda/api/views.py [hunk 4]
```diff
@@ -233,3 +237,93 @@ def post(self, request, *args, **kwargs):
...
+    permission_classes = (permissions.IsAuthenticated & CanPerformTasks,)
+    serializer_class = SetTaskAssigneeSerializer
+
+    def _get_task(self, task_id: str) -> Task:
+        task = get_task(task_id, check_history=False)
+        if not task:
+            raise exceptions.ParseError(
+                _("The task with given task ID does not exist (anymore).")
+            )
+        return task
```
Comment (sergei-maertens) [inline backend/src/zac/camunda/api/views.py:252]: Would put this on the serializer as a validation step.

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**